components/script/dom/htmlformelement.rs

```diff
@@ -302,17 +300,18 @@ impl HTMLFormElement {
 
             // Step 5
             // https://tools.ietf.org/html/rfc7578#section-4
-            result.push_str(&*format!("\r\n--{}\r\n", boundary));
+            let mut boundary_bytes = format!("--{}\r\n", boundary).into_bytes();
```

Comment:
I used WireShark and a node.js simple server to verify that dropping the `\r\n` will make following content correctly interpreted. This behaviour is different from the [RFC 7585](https://tools.ietf.org/pdf/rfc7578.pdf) 4.1 I guess, but it just works :$.